packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -32,9 +35,13 @@ export const App = ({
               <Clear path="/clear/*" />
               <CookiesDisabled path="/cookies_disabled/*" />
               <ResetPassword path="/reset_password/*" />
+              <Legal path="/legal/*" />
+              <LegalTerms path="/legal/terms/*" />
+              <LegalTerms path=":locale/legal/terms/*" />
+              <LegalPrivacy path="/legal/privacy/*" />
+              <LegalPrivacy path=":locale/legal/privacy/*" />
```

Comment:
Do the `:locale/` paths need a leading `/`?

I did a quick grep on this PR locally and it looks like these are the only two paths that start with a `:` so :shrug:
```sh
git grep 'path=":' | cat
packages/fxa-settings/src/components/App/index.tsx:              <LegalTerms path=":locale/legal/terms/*" />
packages/fxa-settings/src/components/App/index.tsx:              <LegalPrivacy path=":locale/legal/privacy/*" />
```